website/content/commands/intention/create.mdx

```diff
@@ -65,3 +65,11 @@ Create intentions from a directory using shell expansion:
 ```shell-session
 $ consul intention create -file intentions/*.json
 ```
+
+<EnterpriseAlert inline /> Create an intention from any service in any namespace
+to the assets-public service in the default namespace:
+
+```shell-session
+# Wrap "*/*" in quotes to prevent wildcard expansion by the shell
```

Comment:
```suggestion
# Wrap "*/*" in quotation marks to prevent wildcard expansion by the shell
```
Perhaps just a nit, but the _quote_ is the stuff between the _quotation marks_.